programmers/수박수박수박수.js

```diff
@@ -0,0 +1,13 @@
+let answer = [];
+
+function solution(n) {
+    for (let i = 1; i <= n; i++) {
+        if (i % 2 === 0) {
+            answer += ("박");
+        }
+        else {
+            answer += ("수");
+        }
+    }
+    return answer;
+}
```

Comment:
1. 전역 변수 선언 지양해주시고 answer은 배열인데 +연산자로 문자열을 넣으셨네요..? +연산자의 자동 형변환을 의도하신건가요..?
2. answer를 배열이 아닌 문자열로 선언하였습니다.
```suggestion
function solution(n) {
    let answer = "";
    
    for (let i = 1; i <= n; i++) {
        if (i % 2 === 0) answer += "박";
        else answer += "수";
    }
    return answer;
}
```
for문에서 i를 1부터 시작한 이유는 알겠는데 이런식으로 하면 좀 더 간결하게 코드 작성이 가능함
```js
function solution(n) {
    let answer = "";
    
    for (let i = 0; i < n; i++) {
        if (i % 2) answer += "박";
        else answer += "수";
    }
    return answer;
}
```